resume-builder/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "resume-builder",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
I would put the testing libraries into dev dependencies to reduce the size of the build files for deployment.